src/core/execute.c

```diff
@@ -943,30 +947,28 @@ static int ask_for_confirmation(const ExecContext *context, const char *vc, Unit
                         r = 1;
                         break;
                 case 'D':
-                        unit_dump(u, stdout, "  ");
+                        printf("Dumping units no longer supported, use 'systemctl show %s'.\n",
+                               params->id);
```

Comment:
The other one doesn't take parameters so it can be done, but this one would need to pass the unit, not sure how to do that given, even if I add a new rtsig, it only takes integers, right?

Reply:
maybe don't dump the unit then, but the exec request data? i.e. ExecContext/ExecParameters/… i.e. the stuff you got passed.